script.js

```diff
@@ -1,4 +1,134 @@
+      allDiceRoll = [];
+      currentDice = 1;
+      return myOutputValue;
+    }
+    if (mode == `select order`) {
+      console.log(diceNumber, allDiceRoll);
+      allValues.push(Number(input));
+      console.log(allValues);
+      myOutputValue = selectOrderMessage(playerNumber, input);
+      mode = `roll dice`;
+      playerNumber = playerNumber + 1;
+    }
+  }
+  if (winCondition == `highest value wins`) {
+    return `The winner is player ${
+      locationOfHighestValue(allValues) + 1
+    } with a value of ${findHighestValueInArray(allValues)}`;
+  }
+  if (winCondition == `lowest value wins`) {
+    return `The winner is player ${
+      locationOfLowestValue(allValues) + 1
+    } with a value of ${findLowestValueInArray(allValues)}`;
+  }
+
+  allValues = [];
+  playerNumber = 1;
+  console.log(locationOfHighestValue(allValues));
 };
+//dice roll function
+var diceRoll = function () {
+  var randomNumber = Math.random() * 6;
+  var randomInteger = Math.ceil(randomNumber);
+  return randomInteger;
+};
+
+//roll dice message
+diceRollMessage = function (playerNumber) {
+  var message = `Player ${playerNumber}: Your dice rolls are ${allDiceRoll}. Please key in your dice roll in the order you want.(eg if your dice roll are 2 and 3, please type in 23/32). Remember, ${winCondition}!`;
+  return message;
+};
+
+//select order function
+
+selectOrderMessage = function (playerNumber, input) {
+  var message = `Player ${playerNumber}: Your value is ${input}. Player ${
+    playerNumber + 1
+  }, please click submit to roll your dice.`;
+  return message;
+};
+//instructions function
+instruction = function () {
+  var message = ` By default, the number of players and number of dice that will be rolled has been set to 2.<br>The rules also follow the standard beat that. <br>If you would like to change any of these, please follow the instructions below.(if the changes made are successful, a message will appear)
+    <br>To change the number of players: Type 'player number' and press sumbit. Next input the number of players you want before pressing submit again. 
+    <br>To change the number of dices rolled: Type 'dice number' and press sumbit. Next input the number of dice you want before pressing submit again.
+    <br> To change the mode: Enter either 'highest value wins' or 'lowest value wins'
+    <br>To select the player/computer to be the one selecting the combination of numbers: Type 'computer' or 'player'`;
+
+  return message;
+};
+
+findHighestValueInArray = function (arrayValue) {
+  highestValue = Math.max(...arrayValue);
+  return highestValue;
+};
+
+findLowestValueInArray = function (arrayValue) {
+  highestValue = Math.min(...arrayValue);
+  return highestValue;
+};
+
+locationOfHighestValue = function (arrayValue) {
+  return arrayValue.indexOf(findHighestValueInArray(arrayValue));
+};
+
+locationOfLowestValue = function (arrayValue) {
+  return arrayValue.indexOf(findLowestValueInArray(arrayValue));
+};
```

Comment:
nice set of functions. Its good to write small functions that focuses on one function